docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,337 @@
+In [Aiven Console](https://console.aiven.io/), you can enable audit logging at the service
+level only. To enable it on a database or for a user, use
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+:::
+
+1. Log in to [Aiven Console](https://console.aiven.io/), and go to your organization
+   \> project > Aiven for PostgreSQL service.
+1. On the **Overview** page of your service, select **Service settings** from the sidebar.
+1. On the **Service settings** page, go to the **Advanced configuration** section
+   and select **Configure**.
+1. In the **Advanced configuration** window, select **Add configuration options**, add
+   the `pgaudit.feature_enabled` parameter, set it to `true`, and select
+   **Save configuration**.
+</TabItem>
+<TabItem value="2" label="Aiven CLI">
+### Enable with Aiven CLI
+
+Use the [Aiven CLI client](/docs/tools/cli) to run the
+[avn service update](/docs/tools/cli/service-cli) command. Update your service by setting
+the `pgaudit.feature_enabled` parameter's value to `true`.
+
+```bash
+avn service update -c pgaudit.feature_enabled=true SERVICE_NAME
+```
+
+:::important
+By default, audit logging does not emit any audit records. To trigger a logging operation
+and start receiving audit records, configure audit logging parameters as detailed in
+[Configure audit logging](#configure-audit-logging).
+:::
+</TabItem>
+<TabItem value="3" label="psql">
+### Enable with psql
+
+:::note
+psql allows for fine-grained enablement of audit logging: on a database, for a user (role),
+or for a database-role combination.
+:::
+
+#### Enable on a database
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   SELECT aiven_extras.set_pgaudit_parameter('log', DATABASE_NAME, 'ddl');
+   ```
+
+#### Enable for a user
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   SELECT aiven_extras.set_pgaudit_role_parameter('log', ROLE_NAME, 'ddl');
+   ```
+</TabItem>
+</Tabs>
+
+## Configure audit logging {#configure-audit-logging}
+
+Configure audit logging by setting
+[its parameters](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme)
+in the [Aiven Console](https://console.aiven.io/), with the [Aiven CLI](/docs/tools/cli),
+or using [psql](https://www.postgresql.org/docs/current/app-psql.html).
+
+:::important
+
+- Advanced configuration of the audit logging feature requires using
+  [psql](https://www.postgresql.org/docs/current/app-psql.html).
+- Any configuration changes take effect only on new connections.
+
+:::
+
+For information on all the audit logging configuration parameters, refer to
+[Settings](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7).
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
+### Configure in Aiven Console
```

Comment:
One question about these subheadings. I'm assuming they were added for navigation (so that they are shown in the list on the right), but it looks like those under *Access in integrated service* aren't shown in the menu. 

Also, the behavior is a bit funky: if I open the page and immediately click "Enable with Aiven CLI" nothing happens, I guess because by default the Aiven Console tab is selected (the "Configure in Aiven Console" link, for example, works). Same if I go to the *Enable audit logging* section and select the CLI tab - the link "Configure with Aiven CLI" doesn't take me to that section. 

Perhaps for now we should leave these headings out like has been done on other pages?